### DISCUSSION TITLE
Som graphics/screenshots of the buttons in Readme

### DISCUSSION THREAD

**Discussion by reev-create** [sButtons/sbuttons#1631] [Ideas]
It would be great and attractive to see some screenshots or previews of the buttons designs in the Readme file

**Comment by shahednasser** [comment]
Thank you for mentioning this @reev-create . I'll create an issue from this idea. Let me know if you're interested in working on it.

**Reply by reev-create** [reply]
I'd love to work on this

**Reply by shahednasser** [reply]
Great! Please comment on issue #1632 so that I can assign you
